app/models/application_search.rb

```diff
@@ -41,10 +41,13 @@ def satisfied?(app, key, value)
   end
 
   def assigned_user_clause(app, value)
```

Comment:
Maybe add a `?` to the end of this as it only returns booleans or things that can be evaluated to booleans.